guides/common/modules/proc_configuring-the-discovery-service.adoc

```diff
@@ -37,17 +37,23 @@ endif::[]
 hosts or virtual machines can be created with incorrect network configurations. On Dell servers, the new naming scheme can be disabled via `biosdevname=1` kernel command line option, for other hardware or virtual machines, the new naming scheme can be completely turned off via `net.ifnames=1`.
 endif::[]
 
+[NOTE]
+====
+ifdef::satellite[]
+The Foreman Discovery image provided with {Project} is based on {EL} 8.
+endif::[]
+ifndef::satellite[]
```

Comment:
```suggestion
ifdef::satellite[]
The Foreman Discovery image provided with {Project} is based on {EL} 8.
endif::[]
ifdef::satellite[]
The Foreman Discovery image provided with {Project} is based on CentOS Stream 8.
endif::[]
ifndef::satellite,orcharhino[]
```

Can you please verify & ACK this @jloeser?

Reply:
You probably mean orcharhino for the second `ifdef` :)

Reply:
Yes :see_no_evil:

Reply:
If we ship official FDI from Foreman then it's also CentOS Stream 8.